### DISCUSSION TITLE
Control/Control activated para background inverse

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1119] [Proposals]
Estamos viendo que la gráfica de facturas no tiene colores Mistica.
Para la gráfica sobre fondo blanco, quedan bien los colores control y control activated, pero para fondo inverse no funcionan.

Necesitaríamos definir como deben comportarse los valores Control y Control Activated para fondo inverse. ¿Nueva variable de color o cogemos otro color de la paleta?

El más similar al que tenemos sería un backgroundSkeletoninverse. Pero no se si constrasta lo suficiente. 
<img width="461" alt="image" src="https://user-images.githubusercontent.com/95479574/224308364-731a2c62-0656-416b-b78b-329da4d2fdc1.png">

🎨 Figma
https://www.figma.com/file/xNUwhH29z3Fre4foNltgtX/CPS_Specs?node-id=8106%3A136042&t=A07LEFcTnHV15dC3-4

**Comment by aweell** [comment]
Hola @MontseTelefonica,

Para el inverse cogería otro color de la paleta

* El `controlActivated` podría pasar a `inverse` (en el spinner por ejemplo esa es la variación definida para cuando va en un fondo inverse).
* El caso del `control`, te paso un par de ejemplos que podrían funcionar, entiendo que el segundo, usando un `applyAlpha` sobre el inverse se acerca al máximo a lo que se busca (habría que validar que esto sea posible)

https://rebrand.ly/adyb23o

**Comment by MontseTelefonica** [comment]
Hello @aweell Creo que con el alpha se ve muy poco, lo descartaría.
He probado con 
Inverse para activated
ButtonPrimaryBackgroundInverse para inactive

Que son colores planos y se asemeja bastante a lo que tenemos ahora mismo:
<img width="852" alt="image" src="https://user-images.githubusercontent.com/95479574/224649427-78725f9a-8b61-45c7-b89c-3c9546424196.png">
Figma->
https://www.figma.com/file/xNUwhH29z3Fre4foNltgtX/CPS_Specs?node-id=8138%3A137597&t=hXpWItwtuyMWqlvY-4

**Reply by aweell** [reply]
Hola @MontseTelefonica, 

Al `applyAlpha` le puedes poner el valor que quieras entre 0 y 1. En el ejemplo use un valor muy bajo pero se puede ajustar (Principalmente porque coincidía con el valor del control en dark). Al final quédate con que permite ponerle alpha a cualquier color de la paleta actual.

Igual `ButtonPrimaryBackgroundInverseSelected` no se comporta como se esperaría en DarkMode. Aquí un ejemplo para que puedas verlo:

https://rebrand.ly/shr75uk

**Reply by MontseTelefonica** [reply]
Cierto. Bien visto.
He planteado color Inverse 40% de alpha para la gráfica en fondo en inverse. Thanks!
https://www.figma.com/file/xNUwhH29z3Fre4foNltgtX/CPS_Specs?node-id=8106%3A136042&t=ywgM631s21E7fJIV-4

**Reply by MontseTelefonica** [reply]
Añado un punto extra a esta discussion e invito a @wyunreal a compartir punto de vista.

Para el hub, se requiere que las barras inactivas se muestren en color distinto al resto de las oBS.
<img width="260" alt="image" src="https://user-images.githubusercontent.com/95479574/224708530-baacd0ff-ffc9-42c1-9733-2cc7f00be1b8.png">

Desde CPS me dicen que ellos no pueden hacer esta excepción en código y que la única forma de hacerlo es crear una var nueva para las barras de estas gráficas y poder darle a cada una el color que se quiera.

**Reply by yceballost** [reply]
@lauraocana esta gráfica no iba a ir con `control` y `controlActivated`? sin ese azul oscuro? por qué se ha vuelto a esto?

**Reply by lauraocana** [reply]
Hola @yceballost, porque es lo que validamos con la OB y Marca en base al ejercicio que hizo Fjord. Como no estamos a tiempo de crear una nueva variable, salimos con tu propuesta de control y controlActivated

**Reply by yceballost** [reply]
Podemos dar esta discussion como cerrada?

**Reply by MontseTelefonica** [reply]
@lauraocana descartamos entonces este cambio definitivamente o lo seguimos viendo como una necesidad post MVP?

**Reply by lauraocana** [reply]
Diría que podemos cerrarla por el momento. Si volviera a salir más adelante porque la OB lo requiere, volvemos a ver @MontseTelefonica ;)
